gnucash/import-export/qif-imp/assistant-qif-import.c

```diff
@@ -3579,6 +3585,11 @@ get_assistant_widgets (QIFImportWindow *wind, GtkBuilder *builder)
 
     wind->window             = GTK_WIDGET(gtk_builder_get_object (builder, "qif_import_assistant"));
     wind->filename_entry     = GTK_WIDGET(gtk_builder_get_object (builder, "qif_filename_entry"));
+    {
+        GtkWidget *enc_con = GTK_WIDGET (gtk_builder_get_object (builder, "encoding_container"));
+        wind->qif_encoding = go_charmap_sel_new (GO_CHARMAP_SEL_TO_UTF8);
+        gtk_container_add (GTK_CONTAINER (enc_con), wind->qif_encoding);
+    }
```

Comment:
Don't make blocks like this, it's very poor style. If you can't do this in the builder file make a helper function:
```C
inline static GtkWidget*
make_encoding_widget (GtkWidget* container)
{
    GtkWidget* encoding_widget = go_charmap_sel_new (GO_CHARMAP_SEL_TO_UTF8);
    gtk_container_add (GTK_CONTAINER (container), encoding_widget);
    return encoding_widget;
}
...
    wind->qif_encoding = make_encoding_widget (gtk_builder_get_object (builder, "encoding_container"));
```